src/catalog/src/system_schema/information_schema/region_peers.rs

```diff
@@ -194,7 +213,8 @@ impl InformationSchemaRegionPeersBuilder {
             let mut table_id_chunks = pin!(table_id_stream.ready_chunks(BATCH_SIZE));
 
             while let Some(table_ids) = table_id_chunks.next().await {
```

Comment:
It's better to rename the `table_id` and `table_id_chunks`. It's the tuple of `(table_id, table_name)` right now.

Reply:
Resolved in [d5b0776](https://github.com/GreptimeTeam/greptimedb/pull/5782/commits/d5b07767973c6f18fa906addc27e2021f493ae34)